testharness/runtest.sh

```diff
@@ -26,102 +28,128 @@ if [ -d output/ ]; then
 fi
 mkdir output
 
+# For local debugging, a much shorter timeout is more appropriate.
+#MYSQLTEST="timeout -k 5s 10s mysqltest --max-connect-retries=2"
 MYSQLTEST="timeout -k 1m 5m $PWD/testharness/mysqltest"
 if [ "$DOLTTESTLINKER" = true ]; then
   MYSQLTEST="timeout -k 1m 5m $PWD/testharness/lib/ld-linux-x86-64.so.2 --library-path $PWD/testharness/lib $PWD/testharness/mysqltest"
 fi
 
-ARG1="$1"
-ARG2="$2"
-if [ -n "$1" ] && [ -z "$2" ]; then
-  FULLSUITENAME="$1"
-  ARG1="${FULLSUITENAME%/*}"
-  ARG2="${FULLSUITENAME#*/}"
-fi
```

Comment:
I like the change that allows for running entire suites, but what about the single-test case where it's passed as "SUITE/TEST" instead of "SUITE TEST"? It's a bit more significant since the output formats tests in that way, so preserving that format for input allows one to just copy and paste it in. Sure it's a small modification, but I think it's worth including.

Reply:
Sure I'll make that change